src/Auth/Models/Throttle.php

```diff
@@ -267,4 +272,4 @@ public function getIsBannedAttribute($banned)
     {
         return (bool) $banned;
     }
-}
+}
```

Comment:
PHP files must terminate with a blank line as per our code style guidelines.

```suggestion
}

```